src/server/middleware.ts

```diff
@@ -137,7 +137,7 @@ export function createCustomManifestMiddleware(): RequestHandler {
                 service.manifestUrl = getProviderUrl(provider);
             }
             if (config) {
-                service.config = JSON.parse(config);
+                service.config = JSON.parse(config!);
```

Comment:
Did TS not like this? It should be able to figure out that `config` exists, due to the surrounding `if`?..

Reply:
We looked at this one together.  I believe the issue was the way we were destructing the values - 

```
            config: null,
            realmName: null,

            // Override with query args
            ...query,
```